alea/runner.py

```diff
@@ -233,7 +398,10 @@ def data_generator(self):
         # generate toydata
         for i_mc in range(self._n_mc):
             if self._toydata_mode == "generate" or self._toydata_mode == "generate_and_store":
-                data = self.statistical_model.generate_data(**self.generate_values)
+                # generate toydata
+                data = self.model.generate_data(**self.generate_values)
+                # set fit guesses as generate values
+                self.model.set_fit_guesses(**self.generate_values)
```

Comment:
This is just the guess of parameters. This will help the optimizer, but will not influence the fitting result.

Reply:
I expect the same. But I was a bit worried because I understood that Zihao observed that the result depended on this.. But I would be surprized if it did much

Reply:
To be honest I am not surprised. The convex optimization is always difficult and may result in a local minimum. Also, we might make inferences about other parameters like ER shift parametric. Maybe can not set them to 0...